src/ILVerify/src/ILImporter.StackValue.cs

```diff
@@ -362,7 +362,7 @@ bool IsBinaryComparable(StackValue src, StackValue dst, ILOpcode op)
                         case StackValueKind.ObjRef:
                             return op == ILOpcode.beq || op == ILOpcode.beq_s ||
                                    op == ILOpcode.bne_un || op == ILOpcode.bne_un_s ||
-                                   op == ILOpcode.ceq;
+                                   op == ILOpcode.ceq || op == ILOpcode.cgt_un;
```

Comment:
Do we also need `bgt_un` here?

The documentation for bgt.un say: `The effect is identical to performing a cgt.un instruction followed by a brtrue`